db/migrate/20190209235051_create_join_table_order_items.rb

```diff
@@ -0,0 +1,12 @@
+class CreateJoinTableOrderItems < ActiveRecord::Migration[5.1]
+  def change
+    create_table :items do |t|
+      t.references :item_id, foreign_key: true
+      t.references :order_id, foreign_key: true
+      t.integer :order_price
```

Comment:
A price is usually a float.  Does ActiveRecord allow for us to use t.float instead of t.integer?  I don't know myself.  Need to look this up.